src/Modules/Workflows/Controllers/WorkflowsList.php

```diff
@@ -99,14 +99,14 @@ public function initialize()
         );
 
         $this->hooks->addAction(
-            'the_title',
+            HooksAbstract::FILTER_THE_TITLE,
```

Comment:
@ojopaul this is an "action", but was named as "filter"

Reply:
Just correcting my comment, the issue is actually in the line before. It should be `$this->hooks->addFilter`.